packages/syft/src/syft/core/adp/data_subject.py

```diff
@@ -149,12 +159,21 @@ def from_string(blob: str) -> DataSubjectGroup:
         return DataSubjectGroup(entity_set)  # type: ignore
 
     def __add__(
-        self, other: Union[DataSubjectGroup, DataSubject, int, float]
+        self,
```

Comment:
Q: I think we this DataSubject Class is deprecated or are we using it again?

Reply:
Yes lets address in a seperate PR.